mmdet/core/utils/dist_utils.py

```diff
@@ -162,6 +162,12 @@ def sync_random_seed(seed=None, device='cuda'):
     because the seed should be identical across all processes
     in the distributed group.
 
+    In distributed sampling, different ranks only need to
+    sample some non-overlapping data in the dataset.
```

Comment:
different ranks should sample non-overlapped data in the dataset.
Therefore, this function is used to make sure that each rank to shuffle the data indices in the same order based on the same seed. Then different ranks could use different indices to select non-overlapped data from the same data list.